src/libs/tools/src/plugindatabase.cpp

```diff
@@ -223,6 +223,21 @@ std::string ModulesPluginDatabase::lookupInfo (PluginSpec const & spec, std::str
 	return plugin->lookupInfo (which);
 }
 
+PluginDatabase::func_t ModulesPluginDatabase::getSymbol (PluginSpec const & spec, std::string const & which) const
+{
+	PluginPtr plugin;
+	try
+	{
+		plugin = impl->modules.load (spec.getName (), spec.getConfig ());
+	}
+	catch (...)
+	{
+		throw;
+	}
```

Comment:
@markus2330 I just copied from `ModulesPluginDatabase::lookupInfo`. Should I change it there too?